content/sips/sip-182.md

```diff
@@ -0,0 +1,161 @@
+        - `c` WETH is unwrapped to ETH and is sent to the user
+        - `c` sETH is burned
+        - `c * burnFeeRate` worth of sETH is swapped to sUSD and sent to the fee pool
+        - `u - c(1+burnFeeRate)` worth of sETH is refunded back to the user
+    - When the user attempts to draw out WETH from the contract by burning `u` sETH and flagging withdrawal in WETH
+      - ✅ Then it succeeds and the following take place:
+        - `c` WETH is sent to the user
+        - `c` sETH is burned
+        - `c * burnFeeRate` worth of sETH is swapped to sUSD and sent to the fee pool
+        - `u - c(1+burnFeeRate)` worth of sETH is refunded back to the user
+  - Given that `u` is strictly lower than `c(1+burnFeeRate)`
+    - When the user attempts to draw out ETH from the contract by burning `u` sETH and flagging withdrawal in ETH
+      - ✅ Then it succeeds and the following take place:
+        - `u (1-burnFeeRate)` worth of WETH is unwrapped to ETH and is sent to the user
+        - `u * burnFeeRate` worth of sETH is swapped to sUSD and sent to the fee pool
+        - `u (1 - burnFeeRate)` sETH is burned
+    - When the user attempts to draw out WETH from the contract by burning `u` sETH and flagging withdrawal in WETH
+      - ✅ Then it succeeds and the following take place:
+        - `u (1-burnFeeRate)` worth of WETH is sent to the user
+        - `u * burnFeeRate` worth of sETH is swapped to sUSD and sent to the fee pool
+        - `u (1 - burnFeeRate)` sETH is burned
+- Given that the contract's holds no WETH
+
+  - When the user attempts draw out ETH or WETH from the contract
+    - ❌ Then the transaction reverts due to the contract being out of WETH
+
+- Given that a user attemps to mint with `WETH` and `msg.value` is greater than zero then the following takes place:
+  - ❌ The transaction reverts due to the user minting with both `WETH` and `ETH`
+
+### Configurable Values (Via SCCP)
+<!--Please list all values configurable via SCCP under this implementation.-->
+There are 3 new values for each Wrapper created by the WrapperFactory:
+
+| Name                    | Setter                       | Description                              |
+| ----------------------- | ---------------------------- | ---------------------------------------- |
+| `wrapperMaxTokenAmount` | `setWrapperMaxTokenAmount`   | Maximum number of tokens in custody of the wrapper |
+| `wrapperMintFeeRate`    | `setWrapperMintFeeRate`      | Portion of synths after minting to be withheld for delivery to fee pool |
+| `wrapperBurnFeeRate`    | `setWrapperBurnFeeRate`      | Portion of synths prior to burning to be withheld for delivery to fee pool |
+
+For all of the above parameters, the corresponding setter or getter function must be supplied with `currencyKey` to indicate which wrapper should be applied the setting.
+
+In addition, `etherWrapperMaxETH`, `etherWrapperMintFeeRate`, and `etherWrapperBurnFeeRate` are removed.
```

Comment:
If you remove these then the current `EtherWrapper` would fail, so they need to stay.